tests/test_no_leader_watch.py

```diff
@@ -826,22 +831,22 @@ def test_punish_small_squad(team_view):
     config = AutoModNoLeaderUserConfig(
         number_of_punishments=2,
         punish_interval_seconds=60,
-        min_squad_players_for_punish=7,
+        min_squad_players_for_punish=6,
         min_server_players_for_punish=0,
         immune_player_level=10,
         immune_roles=[],
     )
-    mod = NoLeaderAutomod(config, None)
+    mod = mod_with_config(config)
 
     watch_status = WatchStatus()
-    player = team_view["allies"]["squads"]["able"]["players"][0]
+    player = team_view["axis"]["squads"]["baker"]["players"][0]
```

Comment:
Any reason that this was changed to baker?

Reply:
The test is supposed to test against a "small" squad (less than 6 players). Able in Allies however is a full squad in our test data (see somewhere at the top). Hence the test failed. Baker in Axis is a squad with 2 players only, which fits perfectly for the intention of the test :)